tests/testthat/test-install.R

```diff
@@ -1,35 +1,51 @@
 context("install")
 
-test_that("test cmdstan installation", {
+test_that("install_cmdstan() successfully installs cmdstan", {
   skip_if_offline()
-  skip_if(on_codecov(), message = "R_COVR")
-
-  if (not_on_cran()) {
-    dir <- dirname(cmdstan_default_path())
-  } else {
-    dir <- tempdir()
-  }
+  dir <- tempdir(check = TRUE)
   expect_message(
     expect_output(
-      install_log <- install_cmdstan(dir = dir, cores = 2, quiet = FALSE, overwrite = TRUE),
-      "Finished installing CmdStan"
+      install_cmdstan(dir = dir, cores = 2, quiet = FALSE, overwrite = TRUE),
+      "Compiling, linking C++ code",
+      fixed = TRUE
     ),
     "CmdStan path set"
   )
-  expect_equal(install_log$status, 0)
 })
 
-test_that("Test cmdstan installation error", {
+test_that("install_cmdstan() errors if installation already exists", {
   skip_if_offline()
   if (not_on_cran()) {
-    dir <- dirname(cmdstan_default_path())
+    dir <- NULL
   } else {
     dir <- tempdir()
   }
   if (dir.exists(file.path(dir, "cmdstan"))) {
-    expect_output(
-      install_cmdstan(dir = dir, cores = 1, quiet = FALSE, overwrite = FALSE),
-      "Please remove or rename the 'cmdstan' folder"
+    expect_warning(
+      install_cmdstan(dir = dir, overwrite = FALSE),
```

Comment:
I think there is a quiet = FALSE missing here? This tests is probably skipped on Travis and that is why its not failing?

Reply:
It was actually a different reason, but fixed now!